### DISCUSSION TITLE
GitHub Copilot Integration - Model Selection

### DISCUSSION THREAD

**Discussion by xcrap** [zed-industries/zed#20203] [Feature Requests]
Now that Copilot is working with multiple models, will this integration would be possible with Zed, I can already select their new models in VSCode Copilot, any idea of this will be extensible to others editors and by others i mean Zed :)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
